.github/workflows/functional-test-noncloud.yaml

```diff
@@ -310,7 +331,31 @@ jobs:
           BICEP_RECIPE_TAG_VERSION: ${{ env.REL_VERSION }}
           TEMP_CERT_DIR: ${{ steps.create-local-registry.outputs.temp-cert-dir }}
           SSL_CERT_FILE: ${{ steps.create-local-registry.outputs.temp-cert-dir }}/certs/${{ env.LOCAL_REGISTRY_SERVER }}/client.crt
+      
+      - name: Publish bicep types 
+        run: | 
+          ./.rad/bin/rad-bicep publish-extension ./hack/bicep-types-radius/generated/index.json --target br:${{ env.LOCAL_REGISTRY_SERVER }}:${{ env.LOCAL_REGISTRY_PORT }}/radius:${{ env.REL_VERSION == 'edge' && 'latest' || env.REL_VERSION }} --force     
```

Comment:
should this be ./.rad/bin/bicep since we downloaded this as bicep file above ?

Reply:
no, we still download the binary to `rad-bicep`  when downloading bicep